salt/modules/junos.py

```diff
@@ -1310,3 +1320,117 @@ def commit_check():
         ret['out'] = False
 
     return ret
+
+
+def get_table(table, file, path=None, target=None, key=None, key_items=None,
+              filters=None, args=None):
+    """
+    Retrieve data from a Junos device using Tables/Views
+
+    Usage:
+
+    .. code-block:: bash
+
+        salt 'device_name' junos.get_table
+
+    Parameters:
+      Required
+        * table:
+          Name of PyEZ Table
+        * file:
+          YAML file that has the table specified in table parameter
+      Optional
+        * path:
+          Path of location of the YAML file.
+          defaults to op directory in jnpr.junos.op
+        * target:
+          if command need to run on FPC, can specify fpc target
+        * key:
+          To overwrite key provided in YAML
+        * key_items:
+          To select only given key items
+        * filters:
+          To select only filter for the dictionary from columns
+        * args:
+          key/value pair which should render Jinja template command
+    """
+
+    conn = __proxy__['junos.conn']()
+    ret = dict()
```

Comment:
Instead of creating a `dict()` here, can you shorten this up and use the literal assignments? It's faster to use  `{}` rather than `dict()` and less memory intensive.